filter_plugins/oo_filters.py

```diff
@@ -881,14 +881,13 @@ def oo_image_tag_to_rpm_version(version, include_dash=False):
         """
         if not isinstance(version, basestring):
             raise errors.AnsibleFilterError("|failed expects a string or unicode")
-        # TODO: Do we need to make this actually convert v1.2.0-rc1 into 1.2.0-0.rc1
-        # We'd need to be really strict about how we build the RPM Version+Release
```

Comment:
I have not gotten to the origin use case much here, I assumed you can specify openshift_release="1.2", and get "v1.2", whatever that is tagged at. If you want to install container tags for an rc you'd specify openshift_image_tag=v1.2.0-rc1-0.